openupgradelib/openupgrade.py

```diff
@@ -799,29 +799,60 @@ def float_to_integer(cr, table, field):
 
 
 def map_values(
-        cr, source_column, target_column, mapping,
-        model=None, table=None, write='sql'):
+        cr, source_column, target_column, mapping, xml_mapping=False,
```

Comment:
Please add new arguments at the end, for keeping compatibility with those that pass arguments positionally